setup.cfg

```diff
@@ -37,6 +40,143 @@ packages =
     zmq
 resources =
     artwork
+
+[py2exe]
+dll_excludes =
+    crypt32.dll,
+    iphlpapi.dll,
+    jvm.dll,
+    kernelbase.dll,
+    mpr.dll,
+    msasn1.dll,
+    msvcm90.dll,
+    msvcp90.dll,
+    msvcr90.dll,
+    nsi.dll,
+    uxtheme.dll,
+    winnsi.dll
+excludes =
+    Cython,
+    ilastik,
+    IPython,
+    matplotlib.backends.backend_cairo,
+    matplotlib.backends.backend_qt*,
+    matplotlib.backends.backend_tk*,
+    pylab,
+    Tkinter,
+    vigra
+includes =
+    h5py,
+    h5py.*,
+    lxml,
+    lxml.*,
+    scipy.io.matlab.streams,
+    scipy.linalg.*,
+    scipy.special,
+    scipy.special.*,
+    scipy.sparse.csgraph._validation,
+    skimage.draw,
+    skimage._shared.geometry,
+    skimage.filters.rank.*,
+    sklearn.*,
+    sklearn.neighbors,
+    sklearn.neighbors.*,
+    sklearn.utils.*,
+    sklearn.utils.sparsetools.*
+packages =
+    cellprofiler,
+    cellprofiler.cpmath,
+    cellprofiler.gui,
+    cellprofiler.icons,
+    cellprofiler.matlab,
+    cellprofiler.modules,
+    cellprofiler.utilities,
+    centrosome,
+    contrib,
+    h5py,
+    imagej,
+    libtiff,
+    lxml
+skip_archive = 1
+
+[py2exe_ilastik]
```

Comment:
And when that happens (I’m an optimist, LOL), is there any reason we’d want to distribute CellProfiler without Ilastik on Windows?